spec/factories.rb

```diff
@@ -0,0 +1,26 @@
+FactoryGirl.define do
+  factory :challenge do
+    title :test
+    description :test
+    input :a
+    output :b
+    diff :c
+    user
+  end
+
+  factory :entry do
+    script :a
+    created_at Time.now.utc
```

Comment:
I will make another PR to convert this across the site. Thank you for pointing it out!